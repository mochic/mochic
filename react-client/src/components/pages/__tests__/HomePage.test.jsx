import React from 'react';
import HomePage from '../HomePage';
import renderer from 'react-test-renderer';

it('renders deep', () => {
  const tree = renderer.create(<HomePage />);
  expect(tree).toMatchSnapshot();
});
