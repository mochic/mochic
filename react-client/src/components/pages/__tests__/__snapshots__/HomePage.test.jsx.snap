// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders deep 1`] = `
<div>
  home page
</div>
`;
